src/cgame/cg_demobackwardscompat.c

```diff
@@ -0,0 +1,33 @@
+#include "cg_local.h"
+
+static qboolean CG_DemoVersionBelow(int major, int minor, int patch)
+{
+	if (cg.demoVersion.major == 0 && cg.demoVersion.minor == 0 && cg.demoVersion.patch == 0)
+	{
+		return qfalse;
+	}
+
+	if (cg.demoVersion.major < major)
+	{
+		return qtrue;
+	}
+	else if (cg.demoVersion.minor < minor)
+	{
+		return qtrue;
+	}
+	else if (cg.demoVersion.patch < patch)
+	{
+		return qtrue;
+	}
+
+	return qfalse;
+}
+
+void CG_DemoBackwardsCompatInit()
+{
+	if (CG_DemoVersionBelow(2, 83, 0))
+	{
+		strcpy(cg.demoBackwardsCompat.characterAnimationGroup, "animations/human_base_2_82_1.anim");
+		strcpy(cg.demoBackwardsCompat.characterAnimationScript, "animations/scripts/human_base_2_82_1.script");
+	}
+}
```

Comment:
`cg_main.c` already contains a function that checks for compatibility, should probably move that into this file

https://github.com/etlegacy/etlegacy/blob/af6404d1b335cf8160b7fdb93a6b5dce6a5d708a/src/cgame/cg_main.c#L3181-L3194